aws/resource_aws_api_gateway_integration.go

```diff
@@ -6,12 +6,14 @@ import (
 	"log"
 	"time"
 
+	"strings"
+
```

Comment:
Do you mind aligning this import with the group above?

Reply:
I didn't notice it at all. vscode did it:smile: